From cbce6d889225fbe771213bc1f1e3f7a5fcb54986 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Mon, 9 May 2016 15:29:19 -0700
Subject: [PATCH] Fix compilation

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 85e2a82..05eb560 100644
--- a/Makefile
+++ b/Makefile
@@ -125,7 +125,7 @@ OBJS := $(OBJS:.cpp=.o)
 OBJS := $(OBJS:.S=.o)
 OBJS := $(OBJS:.s=.o)
 
-CFLAGS += $(DEFINES) $(COMMON_DEFINES) $(INCLUDES)
+CFLAGS += $(DEFINES) $(COMMON_DEFINES) $(INCLUDES) -std=gnu11
 
 
 ifeq ($(platform), theos_ios)
-- 
2.5.0

